customer-config/checkout-config/che.tender.v1-preview.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/extenda/hiiretail-json-schema-registry/master/customer-config/checkout-config/che.tender.v1-preview.json",
+  "title": "Tender API",
+  "description": "The API allows you to define what tenders should be available in a given store\nand their respective behaviour, such as currency id, description,\ntender restriction rules for for specific item category etc.",
+  "type": "array",
+  "items": {
+    "$ref": "#/$defs/Tender"
+  },
+  "$defs": {
+    "Tender": {
+      "description": "Tender includes all the forms of payment that are accepted by the retail store\nin settling sales and other transactions.\n",
+      "type": "object",
+      "required": [
+        "id",
+        "currencyId",
+        "businessUnitGroupId"
+      ],
+      "properties": {
+        "businessUnitGroupId": {
```

Comment:
Should this be businessUnitGroupId or businessUnitId?

Reply:
The property should not be there at all

Reply:
Only the businessUnitGroupId or something more as well?

Reply:
Where do you find the businessUnitGroupId in the tender message then?